lib/galaxy/datatypes/binary.py

```diff
@@ -1907,6 +1907,59 @@ def sniff(self, filename):
             return False
 
 
+class Dcd(Binary):
+    """
+    Class describing a dcd file from the CHARMM molecular simulation program
+
+    >>> from galaxy.datatypes.sniff import get_test_fname
+    >>> fname = get_test_fname('test_glucose_vacuum.dcd')
+    >>> Dcd().sniff(fname)
+    True
+    >>> fname = get_test_fname('interval.interval')
+    >>> Dcd().sniff(fname)
+    False
+    """
+    file_ext = "dcd"
+    edam_format = "format_3842"  # molecular simulation data
```

Comment:
@hmenager does this look good for you?
If so should we also add the same ID for the trr datatype?

Reply:
Link to the ontology - https://www.ebi.ac.uk/ols/ontologies/edam/terms?iri=http%3A%2F%2Fedamontology.org%2Fdata_3842